app/config/config.yml.dist

```diff
@@ -460,3 +460,5 @@ compatibility:
     # Response methods cannot be used on TemplateView objects.
     # Setting this value to false is deprecated.
     template_view: true
+    # Allows specifying a newer version of the setcontent parser
+    setcontent_version: 1
```

Comment:
IMO the key is going away in 4.0 anyways as the new engine will be the only thing available. That's why I chose booleans for the other two keys. I doubt we are going to try to manage more than one storage engine BC breaking change before 4.0.